src/main/java/frc/robot/Robot.java

```diff
@@ -17,32 +27,88 @@ public class Robot extends TimedRobot {
   //Class Initiation
   private final DriveTrain driveTrain = new DriveTrain();
   private final ControlInputs controlInputs = new ControlInputs();
+  private final SensorInputs sensorInputs = new SensorInputs();
+  private Components components = new Components();
+  private final ComponentsControl componentsControl = new ComponentsControl();
+
 
   //Variable Initiation
 
+  //Auto Variable Initiation
+  private String autoSelected;
+  private final String kAutoModeNull = "Do Nothing";
+  private final String kLeave = "Leave";
+  private final String kTurnTest = "Turn Test";
```

Comment:
[Hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation) isn't generally used in Java.